storage-proofs/core/benches/pedersen.rs

```diff
@@ -157,6 +157,7 @@ fn pedersen_circuit_benchmark(c: &mut Criterion) {
     );
 }
 
+#[allow(dead_code)]
```

Comment:
Rather than adding this, I'd put it into `criterion_group!()` at line 220.

Reply:
@vmx - what would that do? will that change the output of some benchmarking tool in some meaningful / surprising way?

Reply:
Good point. It would run more benchmarks. Though I would then also not add the `allow(dead_code)` as I don't expect that it is intentional, so there should be a warning that the code isn't used until someone puts in time to properly fix it.

Reply:
Sounds good. I've removed the `dead_code` as per your suggestion :)